src/kubernetes.js

```diff
@@ -35,4 +35,104 @@ export class KubernetesChaos {
     killNamespace(namespaceName) {
         return this.client.namespaces.delete(namespaceName)
     }
-}
\ No newline at end of file
+
+class JobRunner {
+
+    constructor(client) {
+        this.client = client
+    }
+
+   runJobOnNodes(namespace, jobName, nodes, image, command){
+        let jobs = []
+	for (let i = 0; i < nodes.length; i++) {
+          let name = `${jobName}-${(Math.random() + 1).toString(36).substring(7)}`;
+          console.log(`Creating job ${name} on node ${nodes[i]}`)
+          const job = this.client.jobs.create({
```

Comment:
@dgzlopes Given the complexity of cleaning up jobs and then the corresponding pods (until https://github.com/grafana/xk6-kubernetes/issues/47) is solved, could we instead just start the pods in the given nodes?  I don's see the advantage of running jobs.